public/index.html

```diff
@@ -17,4 +17,4 @@
   <div id="root"></div>
```

Comment:
the favicon.png file belongs to the assets folder. Create an images subfolder and move the file to images folder.

Reply:
No. It must be in the public folder. The home page cannot access data from the src folder.